cli/ballerina-cli/src/main/resources/cli-help/ballerina-new.help

```diff
@@ -34,25 +34,25 @@ DESCRIPTION
        specified, this command creates a package with a 'main' template.
 
 
-       OPTIONS
-            -t <template-name>, --template <template-name>
-                Create a package using a predefined template.
+OPTIONS
```

Comment:
We support templates from central now. Let's add that as well.

Reply:
This was addressed with https://github.com/ballerina-platform/ballerina-lang/pull/34674. We dont need changes to new command help text

Reply:
The content is the same. Its just that the OPTIONS AND EXAMPLES are formatted properly with the correct tab spacing.